doc/cug.tex

```diff
@@ -5114,30 +5141,57 @@ \section{Task Implementation}
 \label{TaskImplementation}
 
 Existing scripts or executables can be used as cylc tasks without any
-modification so long as they return standard exit status on success or
-failure, and do not internally spawn detaching processes
-(see~\ref{DetachingJobs}).
+modification so long as they return standard exit status, i.e. zero on
+success and non-zero for failure, and do not internally spawn detaching
+processes (see~\ref{DetachingJobs}).
+
+\subsection{Task Job Scripts}
+\label{JobScripts}
+
+When a task is ready cylc generates a {\em task job script} that encapsulates
+the runtime settings (environment and scripts) defined for it in the suite.rc
+file. The job script is submitted to run by the {\em job submission method}
```

Comment:
This old terminology should be updated ("job submission method"  --> "batch system")